scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -151,6 +151,26 @@
     tags:
       - create-ardana-input-model
 
+  - name: install patch
+    set_fact:
+        patches: {{ tempest_run_filter}}
+
+  - name: install patch
+    shell: |
+        zypper -n in patch
+    # https://gist.github.com/marcusphi/6791404
+    when: patches is defined and (not patches|trim == '')
+
+  - name: patchit
```

Comment:
Yeah, we need a better description like 'Apply the patches to Ardana source code' or something equivalent